City-of-Helsinki/drupal-helfi-platform-config
# UHF-2286: Kuura Health Chat changes after script update

How to test with existing setup:

1. `composer require drupal/helfi_platform_config:dev-UHF-2286_kuura_health_chat_update && composer require drupal/hdbt:dev-UHF-2286_kuura_health_chat_update`
2. `make drush-cr`
3. Refresh the front page and make sure the chat works as it did before. Also check that it doesn't go over the cookie-banner when it is present on the site.

Also check this PR:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/143
